### ISSUE TITLE
Enable Polls for Amp Story

### ISSUE THREAD

**Issue by ghelapratik** [ampproject/amphtml#15891]
I really love the way Facebook has polls in it's stories. At the moment, AMP Stories is just clickable slides. Having polls can create some sort of interactivity and serve as a data collection point . Check out the screenshot of how Gary vaynerchuk is using it on Facebook Stories

<img width="441" alt="screen shot 2018-06-07 at 2 24 30 pm" src="https://user-images.githubusercontent.com/22929187/41089245-07b87684-6a5f-11e8-9c05-d723c5229e52.png">

**Comment by newmuis** [comment]
Love it!  We are definitely considering ways to improve upon the interactivity in stories; I agree this would be awesome to see.

**Comment by Pateriack** [comment]
"We are definitely considering ways to improve upon the interactivity in stories"

@newmuis any updates on this?

**Comment by newmuis** [comment]
This feature is not one that we have yet; we think we've identified some of the pieces we'd need to put in place to make this work, but it's probably still several months out.

But, for interactivity more generally, we've implemented an experimental feature called "branching" (enabled with the `amp-story-branching` experiment) that allows constructing non-linear stories (e.g. tap targets can jump you to other pages, even if they are not the next page).  This would allow building quiz-like experiences, tables of contents, optional sections of the story, etc.
